File: Csharp/Lab8/Lab8/Lab8/cng/Model/Collections/deprecated/CustomCollections.cs
using System;

namespace Lab8.cng.Model.Collections.deprecated
{
    public interface ICustomCollections<TParam>
    {
        bool Add(TParam param);
        TParam Get(int index);
        int Size();
        bool IsEmpty();
        Object[] CollectionToArray();
        void SetElement(int index, TParam param);

    }
}

File: Java/Desktop/NewLab8/src/by/bntu/fitr/povt/cng/model/collections/deprecated/CustomCollections.java
package by.bntu.fitr.povt.cng.model.collections.deprecated;

public interface CustomCollections <Param> {

    boolean add(Param param);
    Param get(int index);
    int size();
    boolean isEmpty();
    Object[] CollectionToArray();
    void setElement(int index, Param param);



}
